tools/bin/ci_credentials.sh

```diff
@@ -14,7 +14,7 @@ jq --arg v "$GH_INTEGRATION_TEST_CREDS" '.access_token = $v' airbyte-integration
 
 mkdir airbyte-integrations/connectors/source-googleanalytics-singer/secrets/
 echo "$GOOGLE_ANALYTICS_TEST_CREDS" > airbyte-integrations/connectors/source-googleanalytics-singer/secrets/config.json
-echo "GOOGLE_ANALYTICS_TEST_TRACKING_ID" > airbyte-integrations/connectors/source-googleanalytics-singer/secrets/tracker.txt
```

Comment:
@jrhizor was this intentional?

Reply:
Yeah the tracker id is used for the source test but nothing else because it's used to trigger a pageview.

Reply:
Oh, didn't see the `$`. Definitely not intentional. I guess it was "working" by piggybacking off of old page views.

Reply:
Definitely shouldn't impact the functionality of the GA integration itself.